packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -125,6 +127,36 @@ class FileSearchFieldState extends State<FileSearchField>
     _scriptsCache.clear();
   }
 
+  AutoCompleteMatch _createAutoCompleteMatch(String match, String query) {
+    if (query.isEmpty) {
+      return AutoCompleteMatch(match);
+    }
+
+    final autoCompleteResultSegments = <AutoCompleteMatchSegment>[];
+    if (match.contains(query)) {
+      final start = match.indexOf(query);
+      final end = start + query.length;
+      autoCompleteResultSegments.add(AutoCompleteMatchSegment(start, end));
+    } else {
+      final fileName = match.split('/').last;
```

Comment:
can you add a comment as the first line of this else statement that describes this is for a fuzzy match?

Reply:
Good idea, done